docs/reference/rest-api/security/query-api-key.asciidoc

```diff
@@ -455,11 +579,25 @@ The response contains a list of matched API keys along with their sort values:
 <1> The first sort value is creation time, which is displayed in `date_time` <<mapping-date-format,format>> as defined in the request
 <2> The second sort value is the API key name
 
-You can use the following request to retrieve all valid API keys, i.e. not invalidated and not expired:
```

Comment:
I hear you. It's zero effort to reinstate the previous example. But I wanted to combine them because it's too lengthy to explain the concepts in separate code snippets (It's already multiple screens of scrolling to get to this point).
I'll leave it like this, if you're not feeling strongly.